src/Admin/Views/Providers/_ProviderOrganizationScripts.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿<script>
+    function unlinkProvider(providerId, id, orgName) {
+        if (confirm(`Are you sure you want to unlink ${orgName} from this provider?`)) {
```

Comment:
I'm not sure about this bit because we've had to change something similar a while ago https://github.com/bitwarden/server/pull/3218

Reply:
@r-tome Bit confused as I don't think these were part of my change. @cturnbull-bitwarden did you add these?